openhtf/output/web_gui/src/app/station.testheader.ts

```diff
@@ -61,6 +68,17 @@ export class TestHeader implements OnChanges {
   }
 
   /**
+   * Propogates star toggle event back to HistoryHeader component.
```

Comment:
there's a lot of unrelated diffs in this commit. Any chance it was merged with another one on accident?